src/sentry/ingest/consumer/processors.py

```diff
@@ -73,6 +73,7 @@ def process_transaction_no_celery(
         # Delete the event payload from cache since it won't show up in post-processing.
         if cache_key:
             event_processing_store.delete_by_key(cache_key)
+            transaction_processing_store.delete_by_key(cache_key)
```

Comment:
i'd probably put all of the the second write behind a killswitch just in case

Reply:
also if `event_processing_store` is the same as `transaction_processing_store` doesn't this dual write to the same storage?